jax/_src/lax/svd.py

```diff
@@ -43,22 +43,24 @@
 import jax.numpy as jnp
 
 
-@functools.partial(jax.jit, static_argnums=(1, 2))
+@functools.partial(jax.jit, static_argnums=(1, 2, 3))
 def _svd(a: jnp.ndarray,
          is_hermitian: bool,
+         compute_uv: bool,
          max_iterations: int) -> Sequence[jnp.ndarray]:
```

Comment:
Should the output type now be something like
`Union[jnp.ndarray, Sequence[jnp.ndarray]]`

?

In practice type checkers treat `jnp.ndarray` like `Any`, but we don't want to mislead readers.

Reply:
Thanks for the suggestion! Done.